drakrun/drakrun/util.py

```diff
@@ -214,3 +215,11 @@ def graceful_exit(proc: subprocess.Popen):
             proc.kill()
             proc.wait()
             log.error("Process was forceully killed")
+
+
+def file_sha256(filename, blocksize=65536) -> str:
+    snapshot_hash = hashlib.sha256()
+    with open(filename, "rb") as f:
+        for block in iter(lambda: f.read(65536), b""):
```

Comment:
I spotted a typo in `file_sha256`, by adding blocksize parameter I forgot to remove the constant in `f.read` , will quickly fix it

Reply:
:+1: